src/routes/common/transformers.js

```diff
@@ -8,6 +8,7 @@ const transformArticleResponse = (article) => ({
 	createdOn: article.createdAt,
 	articleId: article.id,
 	likesCount : article.likesCount,
+	flagsCount : article.flagsCount,
```

Comment:
not sure the issue mentioned anything about returning this

Reply:
could you give feedback on this discussion?

Reply:
@beblicarl

Reply:
I honestly thought I took it out per your request, I'm surprised it's still there

Reply:
there wasn't a request though, just made a comment. So its mostly to know why it was added if anything

Reply:
and i see its taken out from the specific file like you said, but it is in the service query and the documentation. 

I might lack perspective on why it was added. That's why i don't make a comment like 'remove it' and son

Reply:
oh okay, so after your comment, I went through it and tried to understand why it shouldn't be there and then I took it out. So the reason you still see it is that you are reading from an earlier commit

Reply:
so after the comment, i tried to understand why it shouldn't be there hence I took it out.

Reply:
Okay, I am not reading from an earlier commit. I'm just saying the comment didn't specify an action to take. 

Some comments are just like the start of a discussion and not about changing code so I referred to the comment because it was just marked as resolved but still has other related changes like i specified [here](https://github.com/tbf-challenge/teamwork-be/pull/329#discussion_r1030148029)

Reply:
sure, so initially left it there because I thought it was a good thing for the admin to see the actual number of flags a post has. So a post with a higher number of flags will be of higher priority. Now that we are only returning all posts with a flagCount greater than zero and we aren't actually returning the flagCount for the admin to see I thought it would have been better my way. What do you think?

Reply:
@beblicarl, yea makes sense, let's do it then. We should probably let the query also order by the flagsCount then like you mentioned, so it naturally returns them in descending order of number of flags

Reply:
Yes please, I will do that then